package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dreamjs",
-  "version": "0.1.7",
+  "version": "0.1.9",
```

Comment:
Yes, it's author responsibility to change the version but this wasn't done for 0.1.8 so I've decided it to bump it to correct version in this change.